libpod/container_darwin.go

```diff
@@ -0,0 +1,23 @@
+// +build darwin
+
+package libpod
+
+type containerStateDarwin struct {
+}
+
+// containerStateLinux is intentionally left as a blank stub
+type containerStateLinux struct {
+}
+
+// containerStateWindows is intentionally left as a blank stub
+type containerStateWindows struct {
+}
+
+// IsRootless is a stub function that always returns false on darwin
+func IsRootless() bool {
+	return false
+}
+
+// BecomeRootlessInUserNamespace is a stub function that always returns false on darwin
+func BecomeRootlessInUserNamespace() (bool, error) {
```

Comment:
I think these rootless stubs should go into `pkg/rootless` under a `!linux` build tag or some such.

Reply:
done